docs/job-scheduling.md

```diff
@@ -235,7 +235,7 @@ properties:
   of the cluster. By default, each pool's `minShare` is 0.
 
 The pool properties can be set by creating an XML file, similar to `conf/fairscheduler.xml.template`,
-and setting a `spark.scheduler.allocation.file` property in your
+and either setting `fairscheduler.xml` into classpath or a `spark.scheduler.allocation.file` property in your
```

Comment:
super nit:

... and either putting a file named `fairscheduler.xml` on the classpath, or setting `spark.scheduler.allocation.file` ...

Reply:
Addressed.